b2b-sample/libs/shared/data-access/data-access-common-models-util/src/lib/model/user/userUtils.ts

```diff
@@ -36,15 +37,32 @@ export function decodeUser(user: User) : InternalUser {
     };
 }
 
+/**
+ * 
+ * @param email - email
+ * 
+ * @returns set email.
+ */
+export function setEmail(email: string) {
+    const regex = /^DEFAULT\//g;
+
+    email = email.replace(regex, "");
+
+    return email;
+}
+
 /**
  * 
  * @param userName - user name
  * 
  * @returns set username.
  */
 export function setUsername(userName: string) {
-
-    return userName;
+    if(config.BusinessAppConfig.ManagementAPIConfig.UserStore.trim()===""){
```

Comment:
Formatting issue
`if(config.BusinessAppConfig.ManagementAPIConfig.UserStore.trim()===""){` ->
`if (config.BusinessAppConfig.ManagementAPIConfig.UserStore.trim() === "") {`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e238027e3ab758576ae189fd6ec3d1496abb6315